pyubee/__init__.py

```diff
@@ -314,6 +362,11 @@ def _base_url(self):
         return 'http://{}'.format(self.host)
 
     def _get(self, url, **headers):
+        if hasattr(self, 'authenticator') and isinstance(self.authenticator, DigestAuthAuthenticator):
+            # We are using digest auth:
+            response = requests.get(url, timeout=HTTP_REQUEST_TIMEOUT, auth=HTTPDigestAuth(self.username, self.password))
+            return response
+        """Use the rudimentary auth""" 
```

Comment:
This is a doc-comment, usually found at the top of the method. Please put the old comment (`""" Do a HTTP Get."""`) at the top, and convert this to a normal comment, i.e.:
```
# Use the rudimentary auth
```